src/components/StepList/index.tsx

```diff
@@ -0,0 +1,91 @@
+import styled from '@emotion/styled'
+import PropTypes from 'prop-types'
+import React, { ComponentProps, ReactNode } from 'react'
+import Bullet from '../Bullet'
+
+export const Steps = styled.ul`
+  list-style: none;
+  padding-left: ${({ theme }) => theme.space['2']};
+  margin-top: ${({ theme }) => theme.space['3']};
+  text-align: left;
+  font-size: 16px;
+`
+
+export const Step = styled('li', {
+  shouldForwardProp: prop => !['disabled'].includes(prop.toString()),
+})<{ disabled: boolean }>`
+  margin-bottom: 24px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: ${({ theme, disabled }) =>
+    disabled ? theme.colors.neutral.textWeak : theme.colors.neutral.textStrong};
+`
+
+const StyledDiv = styled('div', {
+  shouldForwardProp: prop => !['size'].includes(prop.toString()),
+})<{ size: Sizes }>`
+  flex: 1;
+  margin-left: ${({ theme, size }) =>
```

Comment:
Could we have a margin relative to font-size ?

Reply:
@chambo-e not sure to understand, that's more or less the case? 

```
margin-left: ${({ theme, size }) =>
    size === 'medium' ? theme.space['2'] : theme.space['1']};
  font-size: ${({ size }) => (size === 'medium' ? '24px' : '16px')};
```